pkg/trivy/wrapper.go

```diff
@@ -34,8 +39,8 @@ func NewWrapper(config etc.Trivy) Wrapper {
 	}
 }
 
-func (w *wrapper) Run(imageRef string, auth RegistryAuth, insecureRegistry bool) (report ScanReport, err error) {
-	log.WithField("image_ref", imageRef).Debug("Started scanning")
+func (w *wrapper) Scan(imageRef ImageRef) (report ScanReport, err error) {
```

Comment:
This function doesn't seem to be unit tested. We should add tests for it as it's an exported function. If not should we consider unexporting it? Thoughts?

Reply:
This Scan function touches global state a lot, so it's tricky to unit test it. That's why I wanted to test at least the GetScanCommand part. Anyway, I think it's a good reason to push it more and refactor further to make it testable.